### ISSUE TITLE
no-indistinguishable-colors bug with color functions

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#875]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Throw error if i you use vendor property.

> Which rule, if any, is this issue related to?

e.g. `no-indistinguishable-colors`

> What CSS is needed to reproduce this issue?

```css
.foo {
    color: #000;
    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
}

```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-indistinguishable-colors": true
  }
}
```

> Which version of stylelint are you using?

e.g. `4.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

No errors.

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
TypeError: Cannot read property 'slice' of null
    at parseColorguardWarning (/usr/lib/node_modules/stylelint/dist/rules/no-indistinguishable-colors/index.js:66:53)
    at /usr/lib/node_modules/stylelint/dist/rules/no-indistinguishable-colors/index.js:30:27
    at Array.forEach (native)
    at /usr/lib/node_modules/stylelint/dist/rules/no-indistinguishable-colors/index.js:29:33
    at /usr/lib/node_modules/stylelint/dist/postcssPlugin.js:129:67
    at Array.forEach (native)
    at /usr/lib/node_modules/stylelint/dist/postcssPlugin.js:98:33
```

**Comment by jeddy3** [comment]
Thanks for the report.

It appears to be `rgba(0, 0, 0, 0)`, rather than the prefixed property, that's causing the function that parsers the colorguard warning to trip up.

**Comment by davidtheclark** [comment]
Fixed in d5654c95cb68bda084afa3b4eb2542f96b6fb9d3
